inc/cleantalk-common.php

```diff
@@ -1092,6 +1092,17 @@ function apbct__styles_if_website_hidden()
 
         echo $styles;
     }
+
+    if ( isset($apbct->settings['forms__wc_honeypot']) && $apbct->settings['forms__wc_honeypot'] ) {
+        $styles = "
+		<style>
+		#apbct_wc_honeypot {
```

Comment:
Could it be two of our honeypot fields on one page?
Isn't it better to use CSS class instead?

Reply:
Fixed